TYPO3.Flow/Classes/TYPO3/Flow/Core/ClassLoader.php

```diff
@@ -440,7 +440,9 @@ protected function initializeAutoloadInformation($composerPath, ApplicationConte
         }
 
         if ($context !== null) {
-            $proxyClasses = @include(FLOW_PATH_DATA . 'Temporary/' . (string)$context . '/AvailableProxyClasses.php');
+            // see Environment::createTemporaryDirectory()
+            $temporaryDirectory = FLOW_PATH_DATA . 'Temporary/' . str_replace('/', '/SubContext', (string)$context) . '/';
```

Comment:
Absolutely. I just copied the code from `Environment::createTemporaryDirectory()` for now